querydsl-spatial/src/main/java/com/querydsl/spatial/jts/JTSPolygonExpression.java

```diff
@@ -40,7 +40,7 @@
     private volatile NumberExpression<Integer> numInteriorRing;
 
     @Nullable
-    private volatile JTSLineStringExpression<LineString> exterorRing;
+    private volatile JTSLineStringExpression<LineString> exteriorRing;
```

Comment:
Yes, that and then remove it back again (so only a local change, no need to push comments)
or splitting the commit.